source/lib/predicates/string.ts

```diff
@@ -175,4 +185,24 @@ export class StringPredicate extends Predicate<string> {
 			validator: value => valiDate(value)
 		});
 	}
+
+	/**
+	 * Test a non-empty string to be lowercase. Matching both alphabetical & numbers.
+	 */
+	get lowercase() {
+		return this.addValidator({
+			message: (value, label) => `Expected ${label} to be lowercase, got \`${value}\``,
+			validator: value => value === (value !== '' && value.toLowerCase())
```

Comment:
Would be better to write this as:

```js
validator: value => value.trim() !== '' && value === value.toLowerCase();
```

I also added trimming of spaces as we shouldn't allow a only-whitespace input either.

Reply:
Good call! Should be fixed now